tests/conftest.py

```diff
@@ -1,14 +1,46 @@
+import contextlib
 import json
 import logging
-import os
 from concurrent.futures import ProcessPoolExecutor
 from pathlib import Path
 from unittest import mock
+from unittest.mock import AsyncMock
 
 import pytest
 
 from app.splash_models import SplashResponse
-from core.website_manager import WebsiteData
+
+
+@contextlib.contextmanager
+def lighthouse_mock(score: float = 0.98):
+    """
+    Mock the request to the lighthouse service of the accessibility extractor. This allows to running unittests
```

Comment:
"allows running" or "allows to run", right?

Reply:
fixed